client/src/main/java/edu/snu/nemo/client/JobLauncher.java

```diff
@@ -102,13 +115,10 @@ public static void launchDAG(final DAG dag) {
       if (jobAndDriverConf == null || deployModeConf == null || builtJobConf == null) {
         throw new RuntimeException("Configuration for launching driver is not ready");
       }
-      final String serializedDAG = Base64.getEncoder().encodeToString(SerializationUtils.serialize(dag));
-      final Configuration dagConf = TANG.newConfigurationBuilder()
-          .bindNamedParameter(JobConf.SerializedDAG.class, serializedDAG)
-          .build();
+      serializedDAG = Base64.getEncoder().encodeToString(SerializationUtils.serialize(dag));
       // Launch and wait indefinitely for the job to finish
       final LauncherStatus launcherStatus = DriverLauncher.getLauncher(deployModeConf)
-          .run(Configurations.merge(jobAndDriverConf, dagConf));
+          .run(jobAndDriverConf);
```

Comment:
Is this the part where the launchDAG message is being sent to the driver? I'm a little confused.

Reply:
Nope. LaunchDAG message is sent by the EventHandler registered on `DriverRPCServer`.